packages/accordions/src/elements/Stepper.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useRef,
+  useEffect,
+  forwardRef,
+  HTMLAttributes,
+  ForwardRefExoticComponent,
+  PropsWithoutRef,
+  RefAttributes
+} from 'react';
+import { StyledStepper } from '../styled';
+import { StepperContext } from '../utils';
+import { Step } from './components/Step';
+import { StepLabel } from './components/StepLabel';
+import { StepContent } from './components/StepContent';
+
+interface IStaticStepperExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Step: typeof Step;
+  Label: typeof StepLabel;
+  Content: typeof StepContent;
+}
+
+interface IStepperProps extends HTMLAttributes<HTMLDivElement> {
+  /** Used to show the current step and compute completed steps */
+  activeIndex?: number;
+  /** Displays the Stepper in a horizontal layout */
+  isHorizontal?: boolean;
+}
+
+/**
+ * Accepts all `<div>` attributes and events. Also accepts static properties:
+
+ *  - `Stepper.Step`
+ *  - `Stepper.StepLabel`
+ *  - `Stepper.StepContent`
```

Comment:
Are these "static properties" as stated above? I think of them as sub-components.

Reply:
That sounds good, updated 👍 .